NEWS

```diff
@@ -8,6 +8,8 @@ NEWS - user visible changes				-*- outline -*-
 
   BIT-OF, BIT-TO-CHAR, HEX-OF, HEX-TO-CHAR
 
+** Support of GCOS 7 (Bull) dialect
+
```

Comment:
New entries belong to the end of the section - in this case before "TODO - more to add".